ui/pages/confirmations/components/confirm/info/shared/advanced-details/advanced-details.tsx

```diff
@@ -123,7 +123,11 @@ const HexDetails = () => {
   return (
     <>
       <ConfirmInfoRowDivider />
-      <ConfirmInfoRow label={t('advancedDetailsHexDesc')}>
+      <ConfirmInfoRow
+        label={t('advancedDetailsHexDesc')}
+        copyEnabled
```

Comment:
Can we now delete the copy button below?

Reply:
Button here is like copy link you see in this image:

<img width="339" alt="Screenshot 2024-07-08 at 11 03 47 AM" src="https://github.com/MetaMask/metamask-extension/assets/2182307/4666185b-6a89-41ae-a47b-0270c27ac7b0">

Reply:
Exactly, my understanding was they do the same thing, so we are switching to the copy icon on the row itself?

Reply:
I see, I removed that copy option now.